mne/utils/docs.py

```diff
@@ -2314,8 +2314,9 @@
 """
 flat_common = """
     Reject epochs based on **minimum** peak-to-peak signal amplitude (PTP).
-    Valid **keys** are ``'grad'``, ``'mag'``, ``'eeg'``, ``'eog'``, ``'ecg'``.
-    The **values** are floats that set the minimum acceptable PTP. If the PTP
+    Valid **keys** can be any channel type that is in ``self.info``. Examples 
```

Comment:
Can you think of another way to say this that doesn't make reference to `self`?  Many end-users will not be familiar with what `self` means.  Maybe 
```
...any channel type present in the object.
```
?